server/conf/application.conf

```diff
@@ -673,4 +673,20 @@ predicates_multiple_questions_enabled = ${?PREDICATES_MULTIPLE_QUESTIONS_ENABLED
 feature_flag_overrides_enabled = false
 feature_flag_overrides_enabled = ${?FEATURE_FLAG_OVERRIDES_ENABLED}
 
-
+# esri address services
+# address correction
+esri_address_correction_enabled = true
```

Comment:
suggestion (blocking): default to false

Reply:
include feature flags, but remove the rest for this PR